src/docs/DescriptionExplicit.stories.ts

```diff
@@ -0,0 +1,30 @@
+import type { Meta, StoryObj } from "@storybook/svelte";
+
+import DescriptionComponent from "./DescriptionComponent.svelte";
+
+/**
+ * This description was set above the `const meta` in the regular CSF file.
+ */
+const meta = {
+	title: "Description From Comment/Regular CSF",
+	component: DescriptionComponent,
+	tags: ["autodocs"],
+	argTypes: {
+		useMeta: { table: { disable: true } },
+		useStory: { table: { disable: true } },
+	},
+	parameters: {
+		controls: { disable: true },
+	},
+} satisfies Meta<DescriptionComponent>;
+
+export default meta;
+
+type Story = StoryObj<DescriptionComponent>;
+
+export const FromMeta: Story = { args: { useMeta: true }, tags: ["!dev"] };
```

Comment:
I think you can set `"!dev"` on the meta level for this to work for all stories, as they inherit that.

```suggestion
export const FromMeta: Story = { args: { useMeta: true }, tags: ["!dev"] };
```